weekly-updates/project-lead-meetings/10_28_18.md

```diff
@@ -82,10 +82,15 @@ _Num architects_: 0
 ## helpq
 
 _UPDATES_:
+made some documentation for meteor basics and also how the lcs integrations work
+did research for rewrite. my curent suggestion is firebase.
 
 _QUESTIONS_:
+gangsta's don't ask questions
 
 _TO DO_:
+fix role update behavior.
+redo design document for firebase
 
 _Num architects_: 1-2
```

Comment:
Nice. Re: the firebase, just note that the free tier might not suffice (it affected the apps before). Please check numbers.